### ISSUE TITLE
Duplicate trigger of events

### ISSUE THREAD

**Issue by dariogriffo** [rebus-org/Rebus.Events#3]
I might be doing something wrong but the events are being fired twice in a simple program.
Attached is an example of this using rabbitmq, but I have also tested with InMemoryBus

Example code [here](https://gist.github.com/dariogriffo/001a1119edf7da2cfd83c9975f1baac1)

Output
![2020-03-26_1928](https://user-images.githubusercontent.com/3247183/77683111-3a2d9780-6f98-11ea-9307-40f80c0a0d99.png)

Platform:
```
Windows 10
netcoreapp3.1
```

Packages installed

```
Rebus.Autofac Version 7.0.0
Rebus.Events Version 4.0.1
Rebus.RabbitMq Version 6.1.0
```

**Comment by dariogriffo** [comment]
PR opened for this [here](https://github.com/rebus-org/Rebus.Events/pull/4)

**Comment by dariogriffo** [comment]
I need the ability to call .Events(c...) more than once, since I'm creating an standard component on top of rebus (that will call .Events once and then the consumer of the component can add more events. All of them are aggregated during configuration time and that's how I found the issue.

**Comment by mookid8000** [comment]
Fixed by #4
